### ISSUE TITLE
Nesting an If alongside an element causes key problems

### ISSUE THREAD

**Issue by AlexGilleran** [AlexGilleran/jsx-control-statements#44]
If you have something like this:

```
<If condition={aCondition}>
    <span>Blah</span>

    <If condition={anotherCondition}>
        <span>Other Blah</span>
    </If>
</If>
```

It seems to cause 
```
warning.js:46 Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `FeatureInfoSection`. See https://fb.me/react-warning-keys for more information.
```

Goes away if you do
```
<If condition={aCondition}>
    <span>Blah</span>

    <If condition={anotherCondition}>
        <span key='otherblah'>Other Blah</span>
    </If>
</If>
```

Looks like it'll be an annoying fix :(

**Comment by stoikerty** [comment]
same thing happens with nested `<Choose>`-blocks

**Comment by texttechne** [comment]
Fixed on current master for if-statements and choose-statements, but not released yet.

@AlexGilleran @stoikerty Could you verify the bugfix?

**Comment by AlexGilleran** [comment]
Works great for me, thanks @texttechne :)

**Comment by stoikerty** [comment]
works for me as well @texttechne 
thanks for the fix!

**Comment by AlexGilleran** [comment]
Fixed in v3.1.3! :dancer:
